File: module-3 fundamental of c/topic-5/question_13.cpp
//Write a program in C to remove characters from a string except alphabets//
#include<stdio.h>
#include<string.h>
int main()
{
	char str1[50];
	int i,j;
	
	printf("enter the string:");
	gets(str1);
	
	// loop for termination of special characters//
	for(i=0;str1[i]!='\0';i++)
    {
        while(!((str1[i]>='a'&&str1[i]<='z')||(str1[i]>='A'&&str1[i]<='Z'||str1[i]=='\0')))
        {
            for(j=i;str1[j]!='\0';j++)
            {
                str1[j]=str1[j+1];
            }
            str1[j]='\0';
        }
    }
	
	printf("output after removal of characters:%s",str1);
	return 0;
}

File: module7(3)python/question13.py
"""Write a Python program to convert a list of characters into a string.
"""
def con(s): 
 
    new = "" 

    for i in s: 
        new += i
 
    return new 
     
     
s = ['v','i','b','h','u','t','i'] 
print(con(s)) 